### ISSUE TITLE
With latest changes selecting audio track does not work.

### ISSUE THREAD

**Issue by ffp3** [simonjhogan/PLAYz-Media-for-PLEX#114]
On my LG 2013 previous version crashes on some video and resume. With new version when try to change audio track it give error that can't change. With previous version it change audio track. LG LA6900

**Comment by braulioholtz** [comment]
I also had this problem, but I'm not a two-language movie on my PC now, you've already tested with the "Make-Down Mix DTS audio for the LG Media Link" activated? in Plex Web->Settings, server, transcoder

**Comment by fjorger** [comment]
Yes, it's true the latests builds (even forks) do have the audio switch problem (fail to set audio).
But the original (LG Store) does work fine...
